### ISSUE TITLE
Release: java_tools v11.0

### ISSUE THREAD

**Issue by comius** [bazelbuild/java_tools#38]
This release splits java_tools into platform independent and prebuilt part. Toolchain definitions are moved from java_tools to bazel repository.

**Comment by comius** [comment]
java_tools build pipeline: https://buildkite.com/bazel-trusted/java-tools-binaries-java/builds/161#cb2ffe65-a848-46f2-a215-13019f52add6
PR with update: https://github.com/bazelbuild/bazel/pull/12552
Downstream tests: https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/1789

**Comment by comius** [comment]
No new failures in the downstream, proceeding with release.

**Comment by comius** [comment]
Change log: 
```
$ git log --pretty=format:"%h %an %s" cd3480e...5a91c25 -- src/java_tools/ third_party/ijar/ tools/jdk/BUILD.java_tools
d10013d Ivo List Separate java_tools into platform independent and prebuilt part.
3c9ad2d philwo Separate ijar sources from deployment zip. (#12556)
2d990cf philwo Fix #10127: Remove Python 2 dependency from tools/android.
de2dab0 Ivo List Separate ijar sources from deployment zip.
b17e9e4 Googler Automatic code cleanup.
```

Commit: https://github.com/bazelbuild/bazel/commit/6202271b9b1bdf1b73b1e2ad55f30cf467899005
Release: https://github.com/bazelbuild/java_tools/releases/tag/java_v11.0
